cassiopeia/preprocess/pipeline.py

```diff
@@ -24,7 +24,7 @@
 from cassiopeia.mixins import logger, PreprocessError
 from cassiopeia.mixins.warnings import PreprocessWarning
 from cassiopeia.preprocess import (
-    alignment_utilities,
+    alignment_utilities as a_utils,
```

Comment:
can we keep this as `alignment_utilities`? I find abbreviations like `a_utils` difficult to read, especially if I'm not familiar with the codebase.

Same as below for `map_utils` and `doublet_utils`.